app/mailers/katello/subscription_mailer.rb

```diff
@@ -0,0 +1,29 @@
+module Katello
+  class SubscriptionMailer < ApplicationMailer
+    after_action :prevent_sending_blank_report
+
+    def subscription_expiry(options)
+      user = ::User.find(options[:user])
+
+      mail_notification = MailNotification.find_by(name: :subscriptions_expiring_soon)
```

Comment:
The audit mailer doesn't do a lookup like this and it's using the mail_query field from what I can tell:

https://github.com/theforeman/foreman/blob/develop/app/mailers/audit_mailer.rb

Reply:
@jturel Updated, thanks!